web/client/components/map/cesium/Map.jsx

```diff
@@ -117,7 +117,8 @@ class CesiumMap extends React.Component {
                 : undefined,
             requestRenderMode: true,
             maximumRenderTimeChange: Infinity,
-            skyBox: false
+            skyBox: false,
+            scene3DOnly: true
```

Comment:
please add a comment explaining the reason of this new flag.
We are using cesium implementation only for 3D visualization and there is an error while converting and normalizing the ifc position when scene3DOnly is false. [Here](https://community.cesium.com/t/overlay-country-data-with-geojson/4941/7) similar a similar problem